src/main/java/core/basesyntax/model/Operation.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax.model;

```

Comment:
You can move this enum to package core.basesyntax.enum for example

Reply:
Idea don't let me do this